Plugin won't install

Hi 

Every time i try to install this plugin i get the below errors :(  Any help would be much appreciated.

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! path git
npm ERR! code ENOENT
npm ERR! errno ENOENT
npm ERR! syscall spawn git
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t git://github.com/tillbaks/node-eiscp.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

--- a/README.md
+++ b/README.md
@@ -49,6 +49,7 @@ Example accessory config (needs to be added to the homebridge config.json):
                 "ip_address": "10.0.0.46",
                 "poll_status_interval": "3000",
                 "name": "Receiver",
+                "zone": "main",
                 "default_input": "net",
                 "default_volume": "10",
                 "max_volume": "40",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-onkyo",
-  "version": "0.7.10",
+  "version": "0.7.11",
   "description": "Homebridge plugin for Onkyo Receivers",
   "main": "index.js",
   "scripts": {
@@ -14,8 +14,7 @@
   "dependencies": {
     "eiscp": "git://github.com/tillbaks/node-eiscp.git",
     "math-round": "^1.0.0",
-    "polling-to-event": "^2.0.2",
-    "request": "^2.69.0"
+    "polling-to-event": "^2.0.2"
   },
   "devDependencies": {
     "eslint": "^6.3.0",

